sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/HelloWorld.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We have renamed this file. Make sure to exclude this file and HelloWorldAsync.java before merge to master.

Reply:
Should this PR modify this file at all?

Reply:
Noop